packages/kuma-gui/src/app/zone-egresses/sources.ts

```diff
@@ -21,18 +21,12 @@ export const sources = (api: KumaApi) => {
   return defineSources({
     '/zone-cps/:name/egresses': async (params) => {
       const { name, size, page } = params
+      const filter = name !== '*' ? {
+        [`labels.${'kuma.io/zone'}`]: name,
+      } : undefined
```

Comment:
When we are in standalone mode with only ZoneEgress `name` comes in as `*` (i.e. all zones I don't care which), in this case we don't send the `?filter` param.